From 7245f06953c1b0d6c71daebe55133997e9559bb8 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Tue, 15 Dec 2020 19:57:32 +0800
Subject: [PATCH] ASoC: es7243e: Add terminate entry for of_device_id table

Fixes below build error:
FATAL: sound/soc/codecs/snd-soc-es7243e: struct of_device_id is not terminated with a NULL entry!

Fixes: 02a49287b00a ("ASoC: es7243e: add es7243e I2S adc support")
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: Iaf1a5dbe7ad4534530d87bca80954caecf861323
---
 sound/soc/codecs/es7243e.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/codecs/es7243e.c b/sound/soc/codecs/es7243e.c
index 25760dcaee6c..f71f1ad631dd 100644
--- a/sound/soc/codecs/es7243e.c
+++ b/sound/soc/codecs/es7243e.c
@@ -2599,6 +2599,7 @@ static const struct of_device_id es7243e_dt_ids[] = {
 #if ES7243E_CHANNELS_MAX > 14
 	{.compatible = "ES7243E_MicArray_7",},	//es7243e_7
 #endif
+	{}
 };
 
 MODULE_DEVICE_TABLE(of, es7243e_dt_ids);
-- 
2.35.3

